sim/core/apl_actions_misc.go

```diff
@@ -45,6 +46,12 @@ type APLActionActivateAura struct {
 	aura *Aura
 }
 
+type APLActionActivateAuraWithStacks struct {
```

Comment:
can you re-order the new code so the new APL actions methods are next to each other. Right now it looks like they are around other actions functions and its confusing